Examples/Swift/AppDelegate+CarPlay.swift

```diff
@@ -88,11 +88,11 @@ extension AppDelegate: CarPlayManagerDelegate {
     
     #if canImport(MapboxGeocoder)
     func carPlayManager(_ carPlayManager: CarPlayManager, searchTemplate: CPSearchTemplate, updatedSearchText searchText: String, completionHandler: @escaping ([CPListItem]) -> Void) {
-        return CarPlayManager.searchTemplate(searchTemplate, updatedSearchText: searchText, completionHandler: completionHandler)
+        return CarPlayManager.searchTemplate(carPlayManager, searchTemplate: searchTemplate, updatedSearchText: searchText, completionHandler: completionHandler)
```

Comment:
There is a pretty significant indication here that this should be an instance method on the carPlayManager itself